core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -74,8 +74,10 @@ class LegacyJavascript extends ByteEfficiencyAudit {
 
     for (const script of artifacts.Scripts) {
       const bundle = bundles.find(bundle => bundle.script.scriptId === script.scriptId);
+      console.time('detectLegacyJavaScript');
       const {matches, estimatedByteSavings} =
         detectLegacyJavaScript(script.content ?? '', bundle?.map ?? null);
+      console.timeEnd('detectLegacyJavaScript');
```

Comment:
Is this some leftover debug stuff? If we want to keep it shouldn't we use `lighthouse-logger`?